pkg/core/cmdutil/flagutil/flagset.go

```diff
@@ -30,9 +30,17 @@ func NewFlagSet(cmd *cobra.Command, localizer localize.Localizer) *FlagSet {
 }
 
 // AddOutput adds an output flag to the command
-func (fs *FlagSet) AddOutput(output *string) {
+func (fs *FlagSet) AddOutput(output *string, format *string) {
```

Comment:
Do we want to edit original function or maybe create new one specifically for this use case?

Reply:
```suggestion
func (fs *FlagSet) AddOutput(output *string) {
    AddOutputFormatted(output *string, ? ) 
}

func (fs *FlagSet) AddOutputFormatted(output *string, format *string) {
```